temporian/implementation/numpy_cc/operators/calendar/common.h

```diff
@@ -0,0 +1,51 @@
+#include "absl/time/time.h"
+
+static absl::TimeZone handle_tz(const pybind11::object tz) {
```

Comment:
you mean `tz`?
I'm pretty rusty in c++ so I'd trust your judgement in this, it would prevent the value from being copied so it sounds like a good idea

Reply:
As a rule of thumb, as soon as an object is larger than a two cpu registers, it is better to use a ref or a pointer.